.github/workflows/update-ai-foundry-models.yml

```diff
@@ -0,0 +1,47 @@
+name: Update AI Foundry Models
+
+on:
+  pull_request:
+    branches:
+      - main
+      - 'release/**'
```

Comment:
Do we want this for release branches too or should be manual porting from `main`?